code/operators/operators.cpp

```diff
@@ -7,22 +7,32 @@ class Fraction {
 
 // TODO:operators
 
+
+void printAndCheck(const std::string & what, const FractionStuff::Fraction & result, const FractionStuff::Fraction & target) {
+  const bool passed = result == target;
+  std::cout << std::left << std::setw(40) << what << ": " << (passed ? "PASS" : "** FAIL **") << "    " << result << "\n";
+}
+void printAndCheck(const std::string & what, bool result, bool target) {
+  const bool passed = result == target;
+  std::cout << std::left << std::setw(40) << what << ": " << (passed ? "PASS" : "** FAIL **") << "    " << result << "\n";
+}
```

Comment:
```suggestion
void printAndCheck(const std::string & what, const Fraction & result, const Fraction & expected) {
  const bool passed = result == expected;
  std::cout << std::left << std::setw(40) << what << ": " << (passed ? "PASS" : "** FAIL **") << "    " << expected << "\n";
}
void printAndCheck(const std::string & what, bool result, bool expected) {
  const bool passed = result == expected;
  std::cout << std::left << std::setw(40) << what << ": " << (passed ? "PASS" : "** FAIL **") << "    " << expected<< "\n";
}
```
I would suggest a different name for `target`. Furthermore, where did the namespace `FractionStuff` come from? :)
And given the redundancy here, I would almost consider using a template here.

Reply:
> I would suggest a different name for target. Furthermore, where did the namespace FractionStuff come from? :)

That's good, but what are you suggesting? 🙂 
It came from some playing around with ADL. Of course, that shouldn't be here! Will fix that.

> And given the redundancy here, I would almost consider using a template here.

I thought that at this point, we didn't discuss templates yet. Overloading is fine, though.

Reply:
On formatting: I didn't use anything but vim in this one. clang-format is of course possible, but there's no official config file yet AFAIK.

Reply:
replace `target` by `expected` as I suggested in the diff.